### PR TITLE
Persona showsecondarytext

### LINKED ISSUE #2887: People Picker: Secondary text is missing from results
- __Package version(s)__: 4.48.1

Requested priority: High

#### Describe the issue:
People Picker results no longer show the secondary text (in our case, email address) so it's impossible to know which result to select.

#### Actual behavior:

image

#### Expected behavior:

Secondary text is rendered

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #2887
- [x] Include a change request file using `$ npm run change`

image

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Persona/Persona.tsx [hunk 1]
```diff
@@ -62,7 +63,8 @@ export class Persona extends BaseComponent<IPersonaProps, {}> {
       primaryText,
       imageShouldFadeIn,
       imageShouldStartVisible,
-      size
+      size,
+      showSecondaryText
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Persona/Persona.tsx]: why is this here?

**Comment by dzearing** [comment]
/home/travis/build/OfficeDev/office-ui-fabric-react/packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx
ERROR: 77:7  no-unused-variable  'className' is declared but never used.
ERROR: 81:7  no-unused-variable  'presence' is declared but never used.
/home/travis/build/OfficeDev/office-ui-fabric-react/packages/office-ui-fabric-react/src/components/Persona/Persona.tsx
ERROR: 57:7  no-unused-variable  'showSecondaryText' is declared but never used.
[office-ui-fabric-react 7:21:56 PM] Error: tslint (19.525s): Error: Command failed: node /home/travis/build/OfficeDev/office-ui-fabric-react/scripts/node_modules/tslint/lib/tslint-cli --project /home/travis/build/OfficeDev/office-ui-fabric-react/packages/office-ui-fabric-react/tsconfig.json -t stylish -r /home/travis/build/OfficeDev/office-ui-fabric-react/common/temp/node_modules/tslint-microsoft-contrib

File: packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx [hunk 2]
```diff
@@ -95,7 +93,7 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
     return (
       <div
         { ...divProps }
-        className={ css('ms-Persona', styles.root, className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence as PersonaPresenceEnum], this.props.showSecondaryText ? styles.showSecondaryText : '') }
+        className={ css('ms-Persona-coin') }
```
Comment (erichdev) [inline packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx]: By removing these styles from PersonaCoin, FacePile doesn't render Persona sizes correctly. 
facepile

I believe the issue is that height of image is based on nested styles, e.g. 
```scss
.root.rootIsExtraExtraSmall {
  height: $ms-Persona-sizeXs;
  min-width: $ms-Persona-sizeXs;

  .imageArea,
  .image {
    flex: 0 0 $ms-Persona-sizeXxs;
    height: $ms-Persona-sizeXxs;
    width: $ms-Persona-sizeXxs;
  }
}

Without the `root` style on PersonaCoin, height defaults to `$ms-Persona-sizeMd`

Also, `ms-Persona-coin` is not defined. Did you mean to add styling for it?

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx]: Great catch Erich. I changed .rootIsExtraExtraSmall to not be tied to the root for imageArea, image, initials etc. This way I can also put the class on the coin for instances that it's used on it's own.
